### DISCUSSION TITLE
Pid tune fall

### DISCUSSION THREAD

**Discussion by Devil3dEG** [mriscoc/Ender3V2S1#1311] [Q&A]
- Hi I am using printer run to tune my pid because I got a new fan and thermistor. I run the m303 e0 s220 c3 and I get this error PID Autotune failed! Bad heater id ok P15 B15 any ideas on how to fix?

**Comment by mriscoc** [comment]
Extruder PID commands doesn't work if your firmware has MPC enabled.

**Reply by Devil3dEG** [reply]
How would I turn it off? Sorry if it's a dumb question I have never done anything firmware related except downloading it lol thanks
